### ISSUE TITLE
pxScene2d xcode workspace build error

### ISSUE THREAD

**Issue by muzehyun** [pxscene/pxCore#716]
git a81e5d7b79da2035247f1fd7819522259b3dfda5
Xcode 9.1
```
Undefined symbols for architecture x86_64:
  "WstCompositorSetDecoderHandleCallback(_WstCompositor*, void (*)(_WstCompositor*, void*, unsigned long long), void*)", referenced from:
      pxWayland::createDisplay(rtString) in pxWayland.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by mfiess** [comment]
This will be fixed by pull request 710.

**Comment by mfiess** [comment]
The changes to fix the xcode build issue have now been committed.

**Comment by conniefry** [comment]
Fixed by commit # a98278097f88acebad69f75531430adf19051fac
